pyris/api/app.py

```diff
@@ -86,6 +86,37 @@ def get(self, code):
         return iris
 
 
+@api.route('/city/code/<string:code>')
+class IrisListFromCityCode(Resource):
+    @api.doc(parser=api.parser(),
+             description="Look for all the iris codes in the city matching a city code")
+    def get(self, code):
+        Logger.info("looking for IRIS in the city %s", code)
+        iris=extract.get_iris_list_by_city_code(code)
+        if not iris:
+            api.abort(404, "City code '{}' not found".format(code))
+        return iris
+
+
+@api.route("/city/search/<string:query>")
+class IrisListFromQuery(Resource):
+    @api.doc(parser=api.parser(),
```

Comment:
When you pass an address as a query search, you can use the `address_parser` such as the `IrisFromAddress` Resource.

```python
class IrisFromAddress(Resource):
    @api.doc(parser=address_parser,
             description="Look for an IRIS for a specific address.")
    def get(self):
        args = address_parser.parse_args()
        query = args['q']
```

In that way, the URL looks like `URL/api/city/search?q=avenue montmartre paris`